crates/algorithms/src/lib.rs

```diff
@@ -41,8 +42,9 @@ pub mod point_in_convex_hull;
 /// A K-Dimensional Tree data structure, useful for various geo-spatial computations.
 pub mod kd_tree;
 
+///impl Haversine formula to calc distance between to points and function to calculate bearing between to points
+pub mod haversine;
 /// A Collection of pathfinding algorithms
 pub mod pathfinding;
-
```

Comment:
Please keep a space between each mod, like so(relevant to both the haversine, and pathfinding):
/// <doc>
pub mod <module>
<space>